### PR TITLE
Add samtools parallel mpileup

### PR DESCRIPTION
**Pull request by shiltemann**
Please sign off by admins only
------------------------------

* [ ] Has no duplicate (otherwise merge required)
* [ ] Indentation is correct
* [ ] Tests
  - [ ] Parameters are reasonably covered
  - [ ] Test files are appropriate
* [ ] `<command/>`: file names are escaped
* [ ] Tool requires version/build increase

### CODE REVIEW

File: packages/package_samtools_parallel_mpileup_0_1_19_a/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+categories:
+- Tool Dependency Packages
+description: samtools - optimized for parallel mpileup generation
+long_description: |
+  Tool dependency definition that downloads and compiles version 0.1.19-a of samtools-parallel-mpileup.
+name: package_samtools_parallel_mpileup_0_1_19_a
+owner: yhoogstrate
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/samtools_parallel_mpileup_galaxy_wrapper
```
Comment (shiltemann) [inline packages/package_samtools_parallel_mpileup_0_1_19_a/.shed.yml]: update

File: tools/samtools_parallel_mpileup/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+categories:
+- SAM
+- Variant Analysis
+description: samtools - optimized for parallel mpileup generation
+long_description: |
+  samtools - optimized for parallel mpileup generation
+name: samtools_parallel_mpileup
+owner: yhoogstrate
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/samtools_parallel_mpileup_galaxy_wrapper
```
Comment (shiltemann) [inline tools/samtools_parallel_mpileup/.shed.yml:9]: update and give access

File: tools/samtools_parallel_mpileup/test-data/generate_reads.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
```
Comment (shiltemann) [inline tools/samtools_parallel_mpileup/test-data/generate_reads.py:1]: flake8 this file

File: tools/samtools_parallel_mpileup/samtools-parallel-mpileup.xml [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+    <outputs>
+        <data format="mpileup" name="output" label="${tool.name} on ${', '.join([ str(a.hid)+': '+a.name for a in $alignments ])}" />
+    </outputs>
+    
+    <tests>
+        <test><!-- Use classical samtools -->
+            <param name="alignments" value="example.bam" ftype="bam" />
+            
+            <param name="source_select" value="history" />
+            <param name="reference_genome" value="example.fa" ftypet="fasta" />
```
Comment (shiltemann) [inline tools/samtools_parallel_mpileup/samtools-parallel-mpileup.xml]: s/ftypet/ftype/

File: tools/samtools_parallel_mpileup/samtools-parallel-mpileup.xml [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+            <param name="parameters" value="default" />
+            
+            
+            <output name="output" file="example.mpileup" /> 
+        </test>
+        <test><!-- Use parallelized samtools - @todo replace with sambamba! -->
+            <param name="alignments" value="example.bam" ftype="bam" />
+            
+            <param name="source_select" value="history" />
+            <param name="reference_genome" value="example.fa" ftypet="fasta" />
```
Comment (shiltemann) [inline tools/samtools_parallel_mpileup/samtools-parallel-mpileup.xml]: s/ftypet/ftype

**Comment by yhoogstrate** [comment]
this wrapper may rest in peace, sambamba can be used instead

**Pull request closed without merging**